.github/workflows/release.yaml

```diff
@@ -38,7 +38,7 @@ jobs:
         id: changesets
         uses: changesets/action@v1
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.RELEASE_WORKFLOW_PAT }}
```

Comment:
- [ ] Create this personal access token for [swisspost-bot](https://github.com/swisspost-bot)

Reply:
Did you do this already?

Reply:
In the end we decided to try with SWISSPOSTDEVS_ACCESS_TOKEN